WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductVariationSelectorViewModel.swift

```diff
@@ -159,10 +159,19 @@ final class ProductVariationSelectorViewModel: ObservableObject {
             return
         }
 
-        if let onVariationSelected = onVariationSelected {
-            onVariationSelected(selectedVariation, parentProduct)
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1) {
```

Comment:
Same comment regarding adding config instead of checking feature flag directly on the shared view.

Reply:
Changed! https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/429403c061684afce086b72cdd514c3004274728